Error using copyfile in MATLAB

```
[[NON-XML-CHAR-0x8]Warning: RigidBodyManipulator: velocity limits are not supported yet][NON-XML-CHAR-0x8] 
[[NON-XML-CHAR-0x8]> In WarningManager/warnOnce (line 21)
  In RigidBodyManipulator/addRobotFromURDF>parseJoint (line 357)
  In RigidBodyManipulator/addRobotFromURDF>parseRobot (line 105)
  In RigidBodyManipulator/addRobotFromURDF (line 55)
  In RigidBodyManipulator (line 75)
  In runPassive (line 12)][NON-XML-CHAR-0x8] 
Constructing a new airfoil
Error using copyfile
The process cannot access the file because it is being used by another process.


Error in RigidBodyWing/onCompile (line 137)
          copyfile(which(profile),fullfile(tempdir,[filename,fileext]));

Error in RigidBodyManipulator/compile (line 720)
        [new_element, model] = model.force{i}.onCompile(model);

Error in RigidBodyManipulator/addRobotFromURDF (line 60)
model = compile(model);  % ideally this would happen on entry into any function...

Error in RigidBodyManipulator (line 75)
          obj = addRobotFromURDF(obj,filename,zeros(3,1),zeros(3,1),options);

Error in runPassive (line 12)
p = RigidBodyManipulator('Plane.URDF', options);

 

CMake Error at C:/projects/Agzs6sMa/drake/cmake/MatlabTestsRedirect.cmake:108 (message):
  [MATLAB] TEST FAILED Matlab returned 1

```
See, e.g., https://drake-cdash.csail.mit.edu/testDetails.php?test=727176&build=24472. Right now this is just on Windows.

--- a/drake/systems/plants/@RigidBodyWing/RigidBodyWing.m
+++ b/drake/systems/plants/@RigidBodyWing/RigidBodyWing.m
@@ -133,9 +133,9 @@ classdef RigidBodyWing < RigidBodyForceElement
           % xfoil cannot handle long filenames (see bug 1734), so
           % copy the profile to the tmp directory and call that instead.
           % yuck.
-          [~,filename,fileext] = fileparts(profile);
-          copyfile(which(profile),fullfile(tempdir,[filename,fileext]));
-          profile = fullfile(tempdir,[filename,fileext]);
+          newname = [tempname,'.dat'];
+          copyfile(which(profile),newname);
+          profile = newname;
 
           avlprofile = strcat('AFILE', '\n', profile);
           xfoilprofile = strcat('LOAD', '\n', profile);

